mods/tuxemon/maps/Azure_Town.tmx

```diff
@@ -0,0 +1,278 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
new assets such as maps, images, audio, etc should be all lower case filenames.  on some platforms, mixing the case can cause problems.